config.json.ctmpl

```diff
@@ -5,6 +5,7 @@
 {
     "dataObjectResolutionHost": {{if eq $runContext "fiab"}}"wb-mock-drs-dev.storage.googleapis.com"{{else if eq $environment "prod"}}"gen3.biodatacatalyst.nhlbi.nih.gov"{{else}}"staging.gen3.biodatacatalyst.nhlbi.nih.gov"{{end}},
     "bondBaseUrl": {{if eq $runContext "fiab"}}"bond-fiab.{{$dnsDomain}}"{{else}}"https://broad-bond-{{$environment}}.appspot.com"{{end}},
-    "samBaseUrl": {{if eq $runContext "fiab"}}"https://sam-fiab.{{$dnsDomain}}"{{else}}"https://sam.dsde-{{$environment}}.broadinstitute.org"{{end}}
+    "samBaseUrl": {{if eq $runContext "fiab"}}"https://sam-fiab.{{$dnsDomain}}"{{else}}"https://sam.dsde-{{$environment}}.broadinstitute.org"{{end}},
+    "jadeDataRepoUrl": {{if eq $environment "prod"}}"jade-terra.datarepo-prod.broadinstitute.org"{{else}}"jade.datarepo-dev.broadinstitute.org"{{end}}
```

Comment:
Is there a reason we need to know the exact jade URLs? `function hasJadeDataRepoHost(someUrl)` can just do a regex match right?

Reply:
If we don't do exact host urls, then with right set of permissions one can talk to prod JDR host in qa/dev env no? Do we want to allow that?

Reply:
Good question. Let's tech talk.

My gut reaction it's better to leave it flexible, versus stopping someone from foot-gunning by sending prod requests in dev. Maybe there's a way we can stay flexible and environmentally segregated.

Reply:
As per the discussion, I have removed the exact matching on host and widened the regex matching. Any data object request which matches the jade data repo regex will be accepted and forwarded to the host mentioned in the request